api/envoy/api/v2/core/grpc_service.proto

```diff
@@ -53,6 +53,11 @@ message GrpcService {
 
         // https://grpc.io/grpc/cpp/namespacegrpc.html#a6beb3ac70ff94bd2ebbd89b8f21d1f61
         google.protobuf.Empty google_default = 2;
+
+        // Use local channel credentials. Currently only UDS is supported.
+        // See
+        // https://github.com/grpc/grpc/blob/782e5c2b8dc6973b6fc0959ba08da0f5b9db2f6f/include/grpcpp/security/credentials.h#L239
+        google.protobuf.Empty local_credentials = 3;
```

Comment:
This should be a message type, which we make empty for now, to allow us to add new fields when local credentials extends beyond UDS without breaking compat.

Reply:
Added a new empty message googleLocalCredentials. Thanks.

Reply:
seems use a dedicated message type hit error https://github.com/envoyproxy/envoy/issues/4578, (I tried to replace ```LocalCredentials``` with ```GoogleDefault```(which is ```types.Empty```), then envoy didn't hit error. 

Could we just use ```types.Empty``` for ```LocalCredentials``` to unblock the integration ?

Reply:
@quanjielin I don't quite get why that would happen. `Empty` is itself just an empty message type.

Reply:
problem solved now(I need to update istio sha to take this PR), thanks !